examples/shell/shell_common/cmd_send.cpp

```diff
@@ -145,16 +143,11 @@ CHIP_ERROR SendMessage(streamer_t * stream)
     gExchangeCtx = gExchangeManager.NewContext({ kTestDeviceNodeId, 0, gAdminId }, &gMockAppDelegate);
     VerifyOrExit(gExchangeCtx != nullptr, err = CHIP_ERROR_NO_MEMORY);
 
-    size = gSendArguments.GetPayloadSize();
-    VerifyOrExit(size <= kMaxPayloadSize, err = CHIP_ERROR_INVALID_MESSAGE_LENGTH);
-
-    requestData = static_cast<char *>(chip::Platform::MemoryAlloc(size));
-    VerifyOrExit(requestData != nullptr, err = CHIP_ERROR_NO_MEMORY);
-
-    snprintf(requestData, size, "CHIP Message");
-    payloadBuf = MessagePacketBuffer::NewWithData(requestData, size);
+    payloadBuf = MessagePacketBuffer::New(gSendArguments.GetPayloadSize());
     VerifyOrExit(!payloadBuf.IsNull(), err = CHIP_ERROR_NO_MEMORY);
 
+    payloadBuf->SetDataLength(gSendArguments.GetPayloadSize());
```

Comment:
Again, we should not be sending uninitialized data.

Reply:
Acked